docs/Building-Tachyon-Master-Branch.md

```diff
@@ -92,7 +103,8 @@ To have the logs output to STDOUT, append the following to the `mvn` command
 
 # Distro Support
 
-To build master against one of the different distros of hadoop, you only need to change the `hadoop.version`.
+To build master against one of the different distros of hadoop, you only need to change the
```

Comment:
"To build Tachyon against...," not specific to master(module?) or master(branch?).